main/main.cpp

```diff
@@ -2857,6 +2857,7 @@ Error Main::setup2() {
 
 		GLOBAL_DEF_BASIC("input_devices/pointing/android/enable_long_press_as_right_click", false);
 		GLOBAL_DEF_BASIC("input_devices/pointing/android/enable_pan_and_scale_gestures", false);
+		GLOBAL_DEF_RST_NOVAL(PropertyInfo(Variant::STRING, "input_devices/pointing/android/rotary_input_scroll_axis", PROPERTY_HINT_ENUM, "Y,X"), "Y");
```

Comment:
I agree the axis value should be an `int` instead of a `string`. This matches the pattern used in the codebase.

As for the labels, I think using `Horizontal, Vertical` would be more clear.

Reply:
`Horizontal,Vertical`. No spacing, it is not trimmed when visualized. Notably, there's really going to ever be just two options so the setting could also be reworked into a toggle.